pkgs/servers/home-assistant/custom-lovelace-modules/frigate-hass-card/package.nix

```diff
@@ -0,0 +1,33 @@
+{
+  lib,
+  stdenv,
+  fetchzip,
+}:
+stdenv.mkDerivation rec {
```

Comment:
Please build the module from source. There are a few examples that use yarn among the other lovelace modules.

Reply:
The card is using yarn berry lock file, so I'm running into errors similar to those mentioned in https://github.com/NixOS/nixpkgs/issues/254369. I might just close this since there's no resolution for this issue yet and I'm hesitant to hack around it.

Could you also review https://github.com/NixOS/nixpkgs/pull/371866 when you get a chance @mweinelt?

Reply:
Meh, in that case, please add a TODO to migrate to the source build once we have support for the yarn berry lockfile format.